packages/stellar-axelar-std/src/interfaces/mod.rs

```diff
@@ -10,11 +10,16 @@ pub use ownable::*;
 pub use pausable::*;
 pub use upgradable::*;
 
-/// Marker trait for interfaces that should not be implemented by using `contractimpl`.
-///
-/// **DO NOT IMPLEMENT THIS MANUALLY!**
-#[doc(hidden)]
-pub trait DeriveOnly {}
+#[macro_export]
+macro_rules! derive_only {
+    () => {
+        /// Marker trait for interfaces that should not be implemented by using `contractimpl`.
+        ///
+        /// **DO NOT IMPLEMENT THIS MANUALLY!**
+        #[doc(hidden)]
+        pub trait DeriveOnly {}
+    };
+}
```

Comment:
If I use the same DeriveOnly trait for all traits that need it, the auto-implementations inside the macros might collide, e.g. when a contract implements both the AxelarExecutableInterface and the InterchainTokenExecutableInterface, so to work around that restriction, each crate that defines such a trait must have their own local DeriveOnly trait